### ISSUE TITLE
Failure to build on arm targets

### ISSUE THREAD

**Issue by e-dant** [nickbabcock/collectd-rust-plugin#112]
Taking any of the examples seems to fail with `--target arm-<anything>-<etc>` (and the aarch64, armvN, etc)

Error is:

```
error[E0308]: mismatched types
   --> /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/collectd-plugin-0.14.0/src/api/mod.rs:218:31
    |
218 |         let list_len = length(list.values_len);
    |                        ------ ^^^^^^^^^^^^^^^ expected `u64`, found `u32`
    |                        |
    |                        arguments to this function are incorrect
    |
note: function defined here
   --> /home/user/.cargo/registry/src/index.crates.io-6f17d22bba15001f/collectd-plugin-0.14.0/src/api/mod.rs:650:8
    |
650 | pub fn length(len: u64) -> usize {
    |        ^^^^^^ --------
help: you can convert a `u32` to a `u64`
    |
218 |         let list_len = length(list.values_len.into());
    |                                              +++++++
```

**Comment by nickbabcock** [comment]
I'll have to dig out an actual arm box, but I wasn't able to reproduce on master.
```
cross build --example write_graphite --target aarch64-unknown-linux-gnu
```
There was a change that may have effected this (https://github.com/nickbabcock/collectd-rust-plugin/pull/95). If you want to try master and if your issue is fixed, then I'll cut a release.

**Comment by e-dant** [comment]
`cargo build --example loadrust --target armv7-unknown-linux-gnueabihf` works on master. 

Linker errors there, though. Haven't investigated them, and a separate issue anyway.

**Comment by e-dant** [comment]
Thank you for your work, by the way

**Comment by nickbabcock** [comment]
I published v0.15.0 Let me know if the issues have been resolved.
